datadog_checks_dev/datadog_checks/dev/tooling/testing.py

```diff
@@ -9,6 +11,7 @@
 
 STYLE_CHECK_ENVS = {'flake8', 'style'}
 STYLE_ENVS = {'flake8', 'style', 'format_style'}
+PYTHON_MAJOR_PATTERN = r'py(\d)'
```

Comment:
```suggestion
PYTHON_MAJOR_PATTERN = r'^py(\d)'
```

Reply:
`re.match` already does this

Reply:
Oh, right, didn't knew that!

> Note that even in MULTILINE mode, re.match() will only match at the beginning of the string and not at the beginning of each line.

https://docs.python.org/3/library/re.html